extensions/iam/oauth2/oauth2-core/README.md

```diff
@@ -4,12 +4,13 @@ This extension provides an `IdentityService` implementation based on the OAuth2
 
 The setting parameters of this extension are listed below:
 
-| Parameter name  | Description  | Mandatory | Default value |
-| :-------------- |:---------------| :---------| :-----------------|
-| `edc.oauth.token.url`  | URL of the authorization server | true | null |
-| `edc.oauth.provider.audience` | Provider audience | false | id of the connector |
-| `edc.oauth.provider.jwks.url` | URL from which well-known public keys of Authorization server can be fetched | false | http://localhost/empty_jwks_url | 
-| `edc.oauth.public.key.alias` | Alias of public associated with client certificate | true | null |
-| `edc.oauth.private.key.alias` | Alias of private key (used to sign the token) | true | null |
-| `edc.oauth.provider.jwks.refresh` | Interval at which public keys are refreshed from Authorization server (in minutes) | false | 5 |
-| `edc.oauth.client.id`  | Public identifier of the client | true | null |
+| Parameter name                    | Description                                                                        | Mandatory | Default value                   |
+|:----------------------------------|:-----------------------------------------------------------------------------------|:----------|:--------------------------------|
+| `edc.oauth.token.url`             | URL of the authorization server                                                    | true      | null                            |
+| `edc.oauth.provider.audience`     | Provider audience                                                                  | false     | id of the connector             |
+| `edc.oauth.provider.jwks.url`     | URL from which well-known public keys of Authorization server can be fetched       | false     | http://localhost/empty_jwks_url | 
+| `edc.oauth.public.key.alias`      | Alias of public associated with client certificate                                 | true      | null                            |
+| `edc.oauth.private.key.alias`     | Alias of private key (used to sign the token)                                      | true      | null                            |
+| `edc.oauth.provider.jwks.refresh` | Interval at which public keys are refreshed from Authorization server (in minutes) | false     | 5                               |
+| `edc.oauth.client.id`             | Public identifier of the client                                                    | true      | null                            |
+| `edc.oauth.validation.nbf.leeway` | Leeway in seconds to remedy clock skew on notBefore claim validation               | false     | 10                              |
```

Comment:
Obvious to us since we know the problem, but perhaps clearly note that the seconds are added to the Connector's time and not subtracted in standard case (unless someone enters a negative int).

something like: `Leeway in seconds added to current time to remedy clock skew on notBefore claim validation`